src/helperFunctions/getCloudinaryImageUrl.ts

```diff
@@ -0,0 +1,18 @@
+const cloudinaryImageUrl = `https://res.cloudinary.com/realdevsquad/image/upload`;
```

Comment:
Can we follow the convention of naming variables with uppercase ? For example -`DUMMY_IMAGE_URL` which you have earlier used. It's more readable.

Reply:
Yeah, That sounds good. implemented. Thanks.